src/docker/Dockerfile

```diff
@@ -30,7 +30,7 @@ FROM ubuntu:14.04
 MAINTAINER Yegor Bugayenko <yegor@teamed.io>
 LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Version="1.0"
 WORKDIR /tmp
-
+ENV OPENJDK_VERSION 7u111-2.6.7-0ubuntu0.14.04.3
 ENV DEBIAN_FRONTEND=noninteractive
 
 # UTF-8 locale
```

Comment:
@amihaiemil I also don't think these locale stuff is necessary - @original-brownbear can you advise?